src/user_account/templates/user_account/base.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %} {{ title }} {% endblock %}
+
+{% block style %}<link rel="stylesheet" href="{% static 'user_account/css/base.css' %}">{% endblock %}
```

Comment:
You replace here `base.css` from global base template. If it is intentional, then OK. In case you want to extend rather than replace, you can do like this:
```
{% block style %}
    {{block.super}}
    <link rel="stylesheet" href="{% static 'user_account/css/base.css' %}">
{% endblock %}
```

Reply:
Thanks!